languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -2525,6 +2525,9 @@ inherit .containing_class_value
     attr (@object.member_pop) pop_symbol = "GUARD:MEMBER"
     edge @object.value -> @object.member_pop
 
+    node @object.class_value
+    node @object.constructor
```

Comment:
I'm pretty sure this is incomplete, but I'm not sure what'll break by their absence. In particular, object literals can't be referenced recursively without a Z combinator, so does it even matter?